### PR TITLE
fix(sap.fhir.model.r4.FHIRListBinding): Invalidate list bindings

### PR DESCRIPTION
**Pull request by AmirthavalliG**
As per suggestion Server-side bindings (e.g. ODataListBinding) are expected to start with a "refresh" eventListBindings "initialize" , fire refresh-event (although not defined in the metadata)

### CODE REVIEW

File: test/qunit/model/FHIRListBinding.integration.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ sap.ui.define(["../utils/TestUtilsIntegration", "../utils/TestUtils"], function
 	}
 
 	function createListBinding(sPath, oContext, aSorters, aFilters, mParameters) {
-		oListBinding = oModel.bindList(sPath, oContext, aSorters, aFilters, mParameters).initialize();
+		oListBinding = oModel.bindList(sPath, oContext, aSorters, aFilters, mParameters);
```
Comment (flovogt) [inline test/qunit/model/FHIRListBinding.integration.js:11]: why changing these lines?

**Review by flovogt — CHANGES_REQUESTED** [review]
Missing tests for this fix

File: test/qunit/model/FHIRListBinding.integration.js (hunk 1 above)
Comment (AmirthavalliG) [reply test/qunit/model/FHIRListBinding.integration.js]: @flovogt : I forgot to send the binding object while overwridding the method. Fixed it now

**Comment by AmirthavalliG** [comment]
> Missing tests for this fix

@flovogt : Done

**Review by flovogt — APPROVED** [review]
LGTM

**Pull request merged**